pallets/liquidity-pools-gateway/src/lib.rs

```diff
@@ -428,20 +574,19 @@ pub mod pallet {
 		type Origin = DomainAddress;
 
 		fn receive(
-			_router_id: T::RouterId,
+			router_id: T::RouterId,
 			origin_address: DomainAddress,
 			message: Vec<u8>,
 		) -> DispatchResult {
-			// TODO handle router ids logic with votes and session_id
```

Comment:
@lemunozm this logic is handled when the message is processed from the queue, just like we do it for outbound.

Reply:
Ok, then you save the `router_id` to process it later, right?

Reply:
Yes, it's part of the inbound gateway message.